web/src/main/webapp/WEB-INF/tags/page.tag

```diff
@@ -0,0 +1,9 @@
+<%@ tag trimDirectiveWhitespaces="true" pageEncoding="utf-8" %>
+<%@ attribute name="pageTitle" required="true" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<tags:header/>
```

Comment:
Usage of <tags:header/> should be inside <tags:master there. You can inspect your page and see that html structure is not what you designed